src/style.css

```
@@ -1,3 +1,82 @@
 body {
+  background-color: green;
   margin: 0;
+  padding: 0;
+}
+
+:root {
+  --main-bg-color: #00ACDC;
+}
+
+html {
+font-family: Roboto, sans-serif;
+}
+
+.header {
+  box-sizing: border-box;
+  background-color: white;
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.nav {
+  margin: 0;
+  padding: 0;
+}
+
+.navbar {
+  display: flex;
+  align-items: center;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+}
+
+.links {
+  display: flex;
+  text-decoration: none;
+  font-size: 12px;
+  color: black;
```

if you make variable for some colors - make it in all cases)
```suggestion
  color: var(--color-black);
```